Age() uses time.Now() directly, breaking the test suite over time

**File:** `recipe/helper_functions.go:262`

`Age()` calls `time.Now()` directly, whereas every other date-aware function uses the package-level injectable `Now` var. Because the test overrides `Now` (to 2021-08-30) but `Age` ignores it, the result drifts every year.

`go test ./...` **fails today**:

```
--- FAIL: TestTransformation_ParseExecute/age_function_calculates_an_age_based_on_a_provided_dob_field
    parse_execute_test.go:758: Dump() = 48 / 22, want 44 / 18
```

**Fix:** use `Now()` (the package var) in `Age` so it is deterministic and testable.

--- a/recipe/helper_functions.go
+++ b/recipe/helper_functions.go
@@ -259,7 +259,7 @@ func Age(dob string) (string, error) {
 	}
 	// Ignoring the error because SmartDate would have already failed on a bad date
 	birthdate, _ := time.Parse(time.RFC3339, normalizedDate)
-	now := time.Now()
+	now := Now()
 	years := now.Year() - birthdate.Year()
 	if now.YearDay() < birthdate.YearDay() {
 		years--

--- a/recipe/parse_execute_test.go
+++ b/recipe/parse_execute_test.go
@@ -710,7 +710,10 @@ func TestTransformation_ParseExecute(t *testing.T) {
 			name:   "age function calculates an age based on a provided dob field",
 			recipe: "1 <- age(1)",
 			input:  "1977-11-07\n2004-02-03\n",
-			want:   "44\n18\n",
+			// Now is pinned to 2021-08-30 (see the Now override below): the
+			// 1977-11-07 birthday has not yet occurred in 2021 (age 43), while
+			// 2004-02-03 already has (age 17).
+			want: "43\n17\n",
 		},
 	}
 

